.github/workflows/test.yml

```diff
@@ -36,17 +36,17 @@ jobs:
       if: runner.os != 'Windows'
       run: >
         cmake -S . -B build -G Ninja
-        --install-prefix $GITHUB_WORKSPACE/dist
-        --toolchain $HOME/conan/conan_paths.cmake
+        --install-prefix ${{ github.workspace }}/dist
+        --toolchain ${{ github.workspace }}/.conan/conan_paths.cmake
         --preset test
 
     - name: Configure CMake build (Windows)
       if: runner.os == 'Windows'
       run: >
         call "${{ matrix.config.vcvars }}"
 
-        cmake -G Ninja -S . -B build --install-prefix %GITHUB_WORKSPACE%\dist
-        --toolchain %USERPROFILE%\conan\conan_paths.cmake -DCMAKE_VERBOSE_MAKEFILE=ON
+        cmake -G Ninja -S . -B build --install-prefix ${{ github.workspace }}\dist
+        --toolchain ${{ github.workspace }}\.conan\conan_paths.cmake -DCMAKE_VERBOSE_MAKEFILE=ON
```

Comment:
[question] Does this work with `/` instead of `\`? As your genius move to switch to yaml subs night mean we could finally get rid of these two independent configure cmake steps?

Reply:
So was the problem before was that we has mismatched separators in the path, so we had to be consistent with what the windows env variable was providing?

Because aye, appears that you're right : https://github.com/elliotcmorris/OpenAssetIO/actions/runs/3298167936/jobs/5439927397

Reply:
A-mazing, nice one. I _think_ we had it before due to the home dir env var differences ie (`%` and the name). 

If you're so inclined and fancy stripping out the duplicate task - that'd be ace, or make an issue and we can do it later if you'd rather avoid the scope creep.

Reply:
There was the VC Vars thing, but I think we have the preamble trick now don't we?

Reply:
Yeah, removing the branching would be really nice

Reply:
Aye, will do, as it's conceptually part of the action to change over the workspace var.

Reply:
Addressed in : [bb2f723](https://github.com/OpenAssetIO/OpenAssetIO/pull/665/commits/bb2f723942344b33201eba3b7510783195d20a64)

You can see [here](https://github.com/OpenAssetIO/OpenAssetIO/actions/runs/3298614794/jobs/5440961991), no more duplicate jobs.

Reply:
Amazing! nice one